src/BuildScriptGeneratorCli/Commands/BuildCommand.cs

```diff
@@ -270,15 +338,15 @@ internal override int Execute(IServiceProvider serviceProvider, IConsole console
                 }
 
                 // Not using IConsole.WriteErrorLine intentionally, to keep the child's error stream intact
```

Comment:
Nit: should we update the `IConsole.WriteErrorLine` part of the comment at this point?

Reply:
thanks for pointing out. that makes sense! I'll update and add more comments later on.